app/renderer/actions/main.ts

```diff
@@ -0,0 +1,11 @@
+import { actionCreator } from "./helpers"
+import * as Actions from "./actionNames"
+import { Transactions } from "app/common/runtimeTypes/storage/transactions"
+import {
+  SaveTransactionsAction,
+} from "./actionTypes"
+
+// Action creator to save wallet
+export const getTransactions = (transactions: Transactions): SaveTransactionsAction => {
```

Comment:
Perhaps a better name for this function could be `saveTransactions`

Reply:
And the logic should be updated to the new type of the transactions `ComputedTransactions`